src/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.csproj

```diff
@@ -17,6 +17,138 @@
     <Compile Include="System\Security\AccessControl\FileSystemRights.cs" />
     <Compile Include="System\Security\AccessControl\FileSystemSecurity.cs" />
     <Compile Include="System\IO\FileSystemAclExtensions.cs" />
+    <Compile Include="..\..\System.IO.FileSystem\src\Microsoft\Win32\SafeHandles\SafeFindHandle.Windows.cs">
```

Comment:
These are all dependencies from the files that we needed from IO.FileSystem.

Reply:
Should we move the files to common instead?

Reply:
If that's the correct procedure, then sure, I'll do it. Are there any cases where the file shouldn't be moved to Common?

Reply:
I moved them. I'd still like to know if there are exceptions to the rule of moving shared files to common.